README.md

```diff
@@ -51,6 +51,19 @@ Any blink-diff custom configuration can be provided so long as the values for `i
 
 By default, we have set [these values](https://github.com/americanexpress/jest-image-snapshot/blob/master/src/diff-snapshot.js#L55) for `thresholdType` and `threshold`.
 
+Custom defaults can be set with a configurable extension.
+
+```javascript
+const { configureToMatchImageSnapshot } = require('jest-image-snapshot');
+
+const blinkDiffConfig = { perceptual: true };
```

Comment:
blinkDiff will no longer be used next release so make sure to name this appropriately.